src/library/basesqltablemodel.cpp

```diff
@@ -782,6 +786,9 @@ QVariant BaseSqlTableModel::data(const QModelIndex& index, int role) const {
                 }
             } else if (column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_REPLAYGAIN)) {
                 value = mixxx::ReplayGain::ratioToString(value.toDouble());
+            } else if (column == fieldIndex(ColumnCache::COLUMN_LIBRARYTABLE_COLOR)) {
+                mixxx::RgbColor::optional_t color = value.isNull() ? std::nullopt : mixxx::RgbColor::optional_t(mixxx::RgbColor(value.toUInt()));
```

Comment:
Isn't this conversion redundant?
QVariant -> std::optional<RgbColor> -> QVariant

Reply:
Yes, I removed it in https://github.com/mixxxdj/mixxx/pull/2470/commits/fd2b5a5855570137cc2856b30e570830b22e9873.